docs/StardustDocs/topics/readSqlDatabases.md

```diff
@@ -106,19 +127,33 @@ Next, import `Kotlin DataFrame` library in the cell below.
 the dataframe library is waiting for a JDBC driver to force classloading.
 
 Find a full example Notebook [here](https://github.com/zaleslaw/KotlinDataFrame-SQL-Examples/blob/master/notebooks/imdb.ipynb).
- 
+
+## Nullability Inference
+
+Each method has an important parameter called `inferNullability`.
+
+By default, this parameter is set to `true`, 
+indicating that the method should inherit the `NOT NULL` constraints from the SQL table definition. 
+However, if you prefer to ignore the SQL constraints
+and determine nullability solely based on the presence of null values in the data, 
+set this parameter to `false`.
+
+When `inferNullability` is set to `false`, 
+the nullability of the column type will be defined based on the number of null values present in the extracted data. 
+If the data contains zero null values, the column will be treated as non-nullable in the Kotlin DataFrame.
```

Comment:
If you're referring to the library "Kotlin DataFrame" by name, no "the" is needed.